README.md

```diff
@@ -352,6 +352,18 @@ The total time in nanoseconds it has taken to load items into cache (cumulative)
 #### indices_in_cache
 For each index that has graphs in the cache, this stat provides the number of graphs that index has and the total graph_memory_usage that index is using in Kilobytes.
 
+#### script_compilations
+The number of times the knn script is compiled. This value should only be 0 or 1 most of the time. However, if the cache containing the compiled scripts is filled, it may cause the script to be recompiled.
+
+#### script_compilation_errors
+The number of errors during script compilation.
+
+#### script_query_requests
+The number of query requests that use the k-NN score script.
+
+#### script_query_errors
```

Comment:
I think these errors are counted at shard level. We could update the documentation accordingly. Also you might want to check if all the above metrics at shard level or index level

Reply:
Yes, they are at shard level. I updated README